### PR TITLE
Changes the pistol sprite of the 88mod/M1911 and replaces the M4A3

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Replaces the Pistol sprites of the m1911/88mod and replaces the m4a3 with a gun by bimmer, 88mod/m1911 by ocelot
Relores the Generic pistol belt a bit, it has the title of m4a3 now, and is no longer magically tgmc equipment.

## Why It's Good For The Game
Sprites that in the style are good. 
Consistency lorewise.
image

## Changelog
:cl:
add: Added new gun sprites, RT-3 by bimmer and 88M4/M1911 by Ocelot/Przyjaciel
del: Removed old gun sprites, git will remember them, nobody else will. M4A3 pistol is gone
tweak: m276 belt made into m4a3 belt
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -70. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Review by Hughgent — APPROVED** [review]

**Review by TheBimmer231 — CHANGES_REQUESTED** [review]
stock is poo, DM me.

File: code/modules/projectiles/guns/pistols.dm [hunk 1]
```diff
@@ -433,11 +422,11 @@
 	scatter = 10
 
 //-------------------------------------------------------
-//VP70 //Not actually the VP70, but it's more or less the same thing. VP70 was the standard sidearm in Aliens though.
+//VP70
 
 /obj/item/weapon/gun/pistol/vp70
 	name = "\improper 88 Mod 4 combat pistol"
```
[the remark this answers is not in the document]
Comment (Pariah919) [reply code/modules/projectiles/guns/pistols.dm:428]: probably not but its just sprite pr for now

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-83). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**